src/handlers/ipcHandlers/generalHandler.js

```diff
@@ -29,4 +30,16 @@ const getCurrentSeason = (e) => {
         console.error(error);
         e.sender.send("General:getCurrentSeasonOK", null);
     })
+}
+
+const getFormula = (e, formulaName) => {
+    let response = general.getFormula(formulaName);
+    console.log(response);
```

Comment:
Do we need to log this?

Reply:
Dang, forgot to remove that after testing